docs/content/latest/manage/backup-restore/point-in-time-recovery.md

```diff
@@ -121,37 +121,26 @@ This feature is in active development. YSQL and YCQL support different features,
 
 ### YSQL limitations
 
-* Currently, you can only recover from the following YSQL operations:
-  * Data changes
-  * CREATE and DROP TABLE
-  * ALTER TABLE - only ADD, DROP, and RENAME COLUMN
-  * CREATE and DROP INDEX
+* For Sequences, restoring to a state before the sequence table was created/dropped doesn't work. This is being tracked by issue [#10249](https://github.com/yugabyte/yugabyte-db/issues/10249).
 
-* The following types of ALTER TABLE aren't supported currently:
-  * ALTER TABLE RENAME TABLE
-  * Other types of ALTER TABLE supported by YSQL but not mentioned above
+* Colocated Tables aren't supported and databases with colocated tables cannot be restored to a previous point in time. Tracked by issue [#8259](https://github.com/yugabyte/yugabyte-db/issues/8259).
```

Comment:
curious, do we actually fail the restore, if it's on a colocated database?

Reply:
I don't think that we have any such checks. We would go ahead with the restore and have unintended consequences (aka shoot ourselves in the foot)